samples/net/openthread/shell/prj.conf

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=4096
```

Comment:
Do we need to increase main stack size given that main thread does literally nothing?

Reply:
Yes, we do. The main stack is also used for driver initialization and the sample doesn't work with the default value `1024`. I checked with the Thread Analyzer and it shows `1688` bytes used. The `coprocessor` sample uses `2560`, but I thought I'd just give it a bit more margin.

Reply:
I have added a comment why the stack increase is required.